lib/adapters/REST/endpoints/app-definition.ts

```diff
@@ -66,3 +86,28 @@ export const del: RestEndpoint<'AppDefinition', 'delete'> = (
 ) => {
   return raw.del(http, getAppDefinitionUrl(params))
 }
+
+export const getInstallationsForOrg: RestEndpoint<'AppDefinition', 'getInstallationsForOrg'> = (
+  http: AxiosInstance,
+  params: GetAppInstallationsForOrgParams & PaginationQueryParams
+) => {
+  if (!params.appDefinitionId) {
+    throw new ValidationError(
+      'appDefinitionId',
+      'Please provide argument of { appDefinitionId: string, organizationId: string } for the getInstallationsForOrg method'
```

Comment:
The two error messages seem inconsistent. I would use the same or group the error into one clause

```
if (!params.appDefinitionId || !params.organizationId) {
...
}
```

Also, while I think having explicit errors makes for a better experience in general, on the other methods we don't do validation. If we want to go down this path, I would do it at least on all the methods in app definitions.

With this change of approach however I would recommend involving @contentful/team-dev-workflows to see if they have opinions or objections

Reply:
@marcolink any thoughts on this? Removing them causes an error to bubble up from the extensibility api, which is way less specific, and probably confusing for users. So, I'm in favor of keeping them, and happy to add error handling to the entire App Definition API

Reply:
@shikaan Marco and I had a chat, and we decided it's more consistent to improve the error message in the Extensibility API. Apropros errors, do we have any sort of overall strategy for error handling?

Reply:
If we make the orgId fall back to an empty string when it is undefined, it bubbles up a decent error from the extensibility API. It's a bad request, which is correct, and it mentions the organizationId. I think that's probably the best outcome we can get if we don't handle the errors directly in this repo.